draft-ietf-moq-transport.md

```diff
@@ -959,10 +983,17 @@ When a relay receives an incoming ANNOUNCE for a given namespace, for
 each active upstream subscription that matches that namespace, it SHOULD send a
 SUBSCRIBE to the publisher that sent the ANNOUNCE.
 
-Object headers carry a short hop-by-hop `Track Alias` that maps to
-the Full Track Name (see {{message-subscribe-ok}}). Relays use the
-`Track Alias` of an incoming Object to identify its track and find
-the active subscribers for that track. Relays MUST forward Objects to
+If a relay receives a Publisher-Chosen Subscription ID in SUBSCRIBE_OK, it
+SHOULD assign this ID to downstream subscriptions for the same track. Since
+subscribers can request tracks from uncoordinated publishers through a single
+relay session, it is not always possible to reuse the upstream PSID.  If there
+is no upstream Publisher-Chosen Subscription ID, or if the upstream PSID is
+already in use downstream for a different track, the relay SHOULD NOT set a
+Publisher-Chosen Subscription ID, and use the Subscriber-Chosen Subscription ID
+instead.
```

Comment:
Do we need this paragraph?

Reply:
I think so - we need to explain that relays should be trying to reuse upstream IDs on the downstream side, and also that this doesn't always work.  This is another reason to make PSID optional - I'd rather just use SSID on the downstream than have the relay invent a new PSID, that itself might conflict with future ones.

Reply:
That means a subscriber can't limit how many PUBLISHes a peer sends it if they're not monotonically increasing.  I'm not sure I want to give up on that so quickly unless we really think it's not useful?